charts/kyverno/templates/networkpolicy.yaml

```diff
@@ -30,4 +30,78 @@ spec:
   ingress:
     - {}
   {{- end }}
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: allow-prometheus
+  namespace: {{ include "kyverno.namespace" . }}
+  labels:
+    app: kyverno
+spec:
+  podSelector:
+    matchLabels:
+      app: kyverno
+  ingress:
+  - from:
+    {{- with .Values.networkPolicy.prometheusNamespaceLabels }}
+    - namespaceSelector:
+        matchLabels:
+            {{ toYaml . | nindent 10 }}
+    {{- end }}
+    {{- with .Values.networkPolicy.prometheusPodLabels }}
+      podSelector:
+        matchLabels:
+            {{ toYaml . | nindent 10 }}
+    {{- end }}
+    ports:
+    # Webhook access
+    - protocol: TCP
+      port: 9443
```

Comment:
I don't think Prometheus needs access to webhook port.  So can probably limit prometheus to just port 8000